dto/service.go

```diff
@@ -292,7 +292,7 @@ type Service struct {
 	SupportedEvidence []string `json:"supportedEvidence,omitempty"`
 
 	// Supported ID documents. This corresponds to "id_documents_supported".
-	SupportedIdentityDocuments []string `json:"supportedIdentityDocuments,omitempty"`
+	SupportedIdentityDocuments []string `json:"supportedDocuments,omitempty"`
```

Comment:
This change is not acceptable. First, that `SupportedIdentityDocuments` generates `supportedDocuments` will confuse developers. Second, data fields for the metadata deprecated by the third implementer's draft of [OpenID Connect for Identity Assurance 1.0](https://openid.net/specs/openid-connect-4-identity-assurance-1_0.html) (OIDC4IDA) should remain there without being removed. See the changes made for version 3.13 of authlete-java-common as a reference ([authlete-java-common CHANGES.md](https://github.com/authlete/authlete-java-common/blob/master/CHANGES.md)).

Note that some metadata of OIDC4IDA will be changed again ([eKYC-IDA PR 87](https://bitbucket.org/openid/ekyc-ida/pull-requests/87)), so it would be better not to touch data fields related to OIDC4IDA until you really need them.

Reply:
Hi Taka, 
the problem is that the client generate an invalid payload to the server if if SupportedIdentityDocuments is populated.
Should I rename the property to "SupportedDocuments"?